airflow/providers/facebook/ads/example_dags/example_ads.py

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that shows how to use FacebookAdsReportToGcsOperator.
+"""
+import os
+
+from facebook_business.adobjects.adsinsights import AdsInsights
+
+from airflow import models
+from airflow.providers.facebook.ads.operators.ads import FacebookAdsReportToGcsOperator
+from airflow.utils.dates import days_ago
+
+# [START howto_GCS_BUCKET_env_variables]
+GCS_BUCKET = os.environ.get("FACEBOOK_ADS_BUCKET", "airflow_bucket_fb")
+GCS_OBJ_PATH = "Temp/this_is_my_report_json.json"
+GCS_CONN_ID = "google_cloud_default"
+# [END howto_GCS_BUCKET_env_variables]
+
+# [START howto_FB_ADS_env_variables]
+FACEBOOK_ADS_CONN_ID = os.environ.get("FACEBOOK_ADS_CONN_ID", "facebook_ads_default")
+FIELDS = [
+    AdsInsights.Field.campaign_name,
+    AdsInsights.Field.campaign_id,
+    AdsInsights.Field.ad_id,
+    AdsInsights.Field.clicks,
+    AdsInsights.Field.impressions,
+]
+PARAMS = {
+    'level': 'ad',
+    'date_preset': 'yesterday'
+}
+# [END howto_FB_ADS_env_variables]
+
+default_args = {"start_date": days_ago(1)}
+
+with models.DAG(
+    "example_fb_operator",
+    default_args=default_args,
+    schedule_interval=None,  # Override to match your needs
+) as dag:
+    # [START howto_FB_ADS_to_gcs_operator]
+    run_operator = FacebookAdsReportToGcsOperator(
+        task_id='run_fetch_data',
+        start_date=days_ago(2),
+        owner='airflow',
+        facebook_ads_conn_id=FACEBOOK_ADS_CONN_ID,
+        bucket_name=GCS_BUCKET,
+        params=PARAMS,
+        fields=FIELDS,
+        gcp_conn_id=GCS_CONN_ID,
+        object_name=GCS_OBJ_PATH,
+    )
```

Comment:
Wouldn't you like to extend this example to a more real case, i.e. add data copy to BigQuery? Many users will use it this way, so testing this case will be helpful.
Here is an example:
https://github.com/apache/airflow/blob/master/airflow/providers/google/cloud/example_dags/example_presto_to_gcs.py
It can also be an interesting task for you, because you will have to get to know BigQuery.

Reply:
Sounds good! Will have a look @mik-laj